experiments/experiment.py

```diff
@@ -157,6 +157,8 @@ def format_data(self, state, full_state=True):
                 data[key] = float(data[value])
             elif isinstance(value, dict):
                 data[key] = self.format_data(value)
+            elif isinstance(value, (list, set)):
```

Comment:
just for generality can we do 
```suggestion
            elif hasattr(value, '__iter__'):
```
this should handle lists, sets, tuples, frozensets, and other weird sequence types, but ignore strings